src/main/java/com/puppycrawl/tools/checkstyle/checks/SuppressWarningsHolder.java

```diff
@@ -64,6 +64,18 @@
  * This will prevent from invocation of the MemberNameCheck:
  * </p>
  * <pre>
+ * &lt;module name=&quot;TreeWalker&quot;&gt;
```

Comment:
Please remove `To prevent {@code FooCheck} violations` block
Users do need real usage and real use cases

Reply:
should I have to remove this 
```
@SuppressWarnings("all")
public void someFunctionWithInvalidStyle() {
  //...
} 
```

Reply:
Keep it, but make violation from two Checks to show that 'all' means all :).

Reply:
> should I have to remove this too
> 
> ```
> @SuppressWarnings("all")
> public void someFunctionWithInvalidStyle() {
>   //...
> } 
> ```

should i change code used in it

Reply:
done